plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointFieldReader.java

```diff
@@ -96,6 +97,21 @@ public long getLong(String fieldName)
         return valueBlock.getLong(row.getUnderlyingFieldPosition(field));
     }
 
+    public OptionalLong getOptionalLong(String fieldName)
```

Comment:
Which test ensures this logic?

Reply:
The `TestDeltaLakeBasic#testReadInCommitTimestampInHistoryTable` tests it, while reading from a Delta Lake history table with and without an existing `inCommitTimestamp`

Reply:
https://github.com/trinodb/trino/blob/04670c4e8cfb4c39e36e1a9aa40c1d9b2ffa5419/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TestCheckpointEntryIterator.java#L618-L623

Seems there isn't a case can hit this logic, I will remove this helper function and leave here a OptionalLong.empty().